charts/ingress-nginx/templates/controller-ingressclass.yaml

```diff
@@ -18,4 +18,22 @@ metadata:
 spec:
   controller: {{ .Values.controller.ingressClassResource.controllerValue }}
   {{ template "ingressClass.parameters" . }}
+{{- if .Values.controller.ingressClassResource.extraNames -}}
```

Comment:
I would put this in another file

Reply:
What's your opinion on the variable naming? I also thought about something like `aliases`, but not sure since this gives the feeling of having the same `IngressClass` with multiple names instead of whole copies of it.

Reply:
updated to an independent file.